core/src/main/web/app/controlpanel/controlpanel-directive.js

```diff
@@ -60,6 +63,17 @@
           return bkMenuPluginManager.getMenus();
         };
 
+        if (bkUtils.isElectron){
+          window.addEventListener('focus', function() {
```

Comment:
> Does registering different handlers for the same event


unless they are identical they will build up

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Multiple_identical_event_listeners

checkout this:
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Memory_issues